content/tools/related-tools.md

```diff
@@ -61,6 +61,7 @@ People have written tools on top of Cucumber. These are the known third-party to
 | [OCCukes](https://github.com/royratcliffe/OCCukes)                                                              | Objective-C wire protocol implementation. Can be used for testing native iOS and Mac apps.                                                                                                                                                                                                   |
 | [Pickle](https://github.com/ianwhite/pickle/tree/master)                                                        | Easy model creation/reference in Cucumber - optionally leveraging your factories/blueprints                                                                                                                                                                                                  |
 | [Pickler](https://github.com/tpope/pickler/tree/master)                                                         | Synchronize user stories in [Pivotal Tracker](http://www.pivotaltracker.com/) with Cucumber Features.                                                                                                                                                                                        |
+| [Pickles Docs](http://www.picklesdoc.com/)                                                                      | takes your Specification (written in Gherkin, with Markdown descriptions) and turns them into an always up-to-date documentation of the current state of your software - in a variety of formats (HTML SPA, Static HTML, JSON, Word .docx, Excel)                                  |
```

Comment:
Thanks for adding this! Two small requests for consistency: 1. Could you please use a capital at the start of the sentence ("Takes" instead of "takes") to be consistent with the other tools. 2. Could you use lower case for terminology in the sentence (so "specification" instead of "Specification"?

Reply:
Applied suggestions